### PR TITLE
Modifications to use the new version of the unicycle planner.

### PR DESCRIPTION
**Pull request by S-Dafarra**
In order to merge this, first https://github.com/robotology/unicycle-footstep-planner/pull/17 should be merged.

### CODE REVIEW

**Comment by S-Dafarra** [comment]
@DanielePucci, @MiladShafiee and me tested this PR on the robot and seems to be working fine. Ready to merge when https://github.com/robotology/unicycle-footstep-planner/pull/17 is merged and @GiulioRomualdi approves.

File: modules/Walking_module/src/TrajectoryGenerator.cpp [hunk 1]
```diff
@@ -106,44 +106,52 @@ bool TrajectoryGenerator::configurePlanner(const yarp::os::Searchable& config)
 
     m_swingLeft = config.check("swingLeft", yarp::os::Value(true)).asBool();
     bool startWithSameFoot = config.check("startAlwaysSameFoot", yarp::os::Value(false)).asBool();
-    bool useMinimumJerkFootTrajectory = config.check("useMinimumJerkFootTrajectory",
-                                                     yarp::os::Value(false)).asBool();
+    m_useMinimumJerk = config.check("useMinimumJerkFootTrajectory",
```
Comment (GiulioRomualdi) [inline modules/Walking_module/src/TrajectoryGenerator.cpp]: Why is `m_useMinimumJerk` an attribute of `TrajectoryGenerator` class?

**Comment by GiulioRomualdi** [comment]
Approved but before merging  robotology/unicycle-footstep-planner#17 has to be merged

**Comment by GiulioRomualdi** [comment]
The robotology/unicycle-footstep-planner#17 was merged. I'm going to merge this PR

**Pull request merged**